docs/integrations/sources/google-drive.md

```diff
@@ -243,6 +243,18 @@ One record will be emitted for each document. Keep in mind that large files can
 
 Before parsing each document, the connector exports Google Document files to Docx format internally. Google Sheets, Google Slides, and drawings are internally exported and parsed by the connector as PDFs.
 
+<HideInUI>
+
+## Usage with airbyte-lib
+
+For document file type streams, make sure the `tesseract` and `pdftotext` libraries are installed. On MacOS, you can install them with `brew install poppler tesseract`.
```

Comment:
Why are we putting `MacOS` specifically here as an example to install those, but no other system?

Reply:
This is just an example, I will remove it from this PR before merging (only the pure implementation of the components will go on, rolling it out to connectors on a separate PR)